docs/gettsim_objects/input_variables.md

```diff
@@ -147,7 +147,7 @@ Type: float
 
 ## `eink_vermietung_m`
 
-Monthly rental income
+Monthly rental income net of maintenance expenses
```

Comment:
```suggestion
Monthly rental income net of deductions.
```

Better not get started enumerating all possibilities...

Reply:
Sounds good. But the other variable description also do not have a period at the end. Should we change it?

Reply:
No worries, all good! Did not check for consistency.